ui_framework/components/color_picker/color_picker.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { ChromePicker } from 'react-color';
+
+export class KuiColorPicker extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showColorSelector: false,
+    };
+  }
+
+  closeColorSelector = () => {
+    this.setState({ showColorSelector: false });
+  };
+
+  toggleColorSelector = () => {
+    this.setState({ showColorSelector: !this.state.showColorSelector });
+  };
+
+  handleColorSelection = (color) => {
+    this.props.onChange(color.hex);
+  };
```

Comment:
Because it's an assignment, not a function, intellij complains without the semi colon.
<img width="375" alt="screen shot 2017-06-14 at 9 00 01 am" src="https://user-images.githubusercontent.com/16563603/27133433-0b24dfe0-50e0-11e7-8269-3bc53ce61c08.png">
So I think it's actually more correct to use the semis in this situation.